int32 linear_velocity_x
int32 linear_velocity_y
int32 linear_velocity_z
int32 angular_velocity_yaw